proto/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package proto
+
+//go:generate protoc --go_out=. --go_opt=module=github.com/veraison/ratsd/proto compositor.proto
```

Comment:
We want to generate the Go stubs for GRPC server as well, I'd recommend the following change
```suggestion
//go:generate protoc --go_out=. --go-grpc_out=. --go_opt=module=github.com/veraison/ratsd/proto compositor.proto
```

Reply:
I'll check this and get back.